packages/gluten-async/gluten-async.0.3.0/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+maintainer: "Antonio Monteiro <anmonteiro@gmail.com>"
+authors: [ "Antonio Monteiro <anmonteiro@gmail.com>" ]
+license: "BSD-3-clause"
+homepage: "https://github.com/anmonteiro/gluten"
+bug-reports: "https://github.com/anmonteiro/gluten/issues"
+dev-repo: "git+https://github.com/anmonteiro/gluten.git"
+doc: "https://anmonteiro.github.io/gluten/"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "1.0"}
+  "gluten" {= version}
+  "faraday-async" {>= "0.7.2"}
```

Comment:
There are currently no version of `faraday-async` compatible with the janestreet suite of packages v0.15

Reply:
I'm not sure what to do. There's https://github.com/inhabitedtype/faraday/pull/76